web/libs/editor/src/components/CurrentEntity/AnnotationHistory.tsx

```diff
@@ -142,7 +142,7 @@ const AnnotationHistoryComponent: FC<any> = ({
   if (shouldShowEmptyState) {
     return (
       <Block name="annotation-history" mod={{ inline, empty: true }}>
-        {sectionHeader && <Elem name="section-head">{sectionHeader}</Elem>}
+        {sectionHeader && <div className="lsf-annotation-history__section-head sr-only">{sectionHeader}</div>}
```

Comment:
This should do it
```suggestion
        {sectionHeader && <Elem name="section-head" rawClassName="sr-only">{sectionHeader}</Elem>}
```
Alternatively you can import `cn` and use it like this `cn('annotation-history').elem('section-head').toString()` then you can pass in sr-only as a classname with it too - this way  you dont need to manually put in the prefix (which could change in the future)

Reply:
This was to be my same feedback above ^.

Reply:
The only thing additional I will mention is that this will hide all sectionHeader content, regardless if there is history or not, which I don't believe we want.

Reply:
just for the context this header displays id of current annotations, not history item

Reply:
That is correct, that is why hiding this ubiquitously seems incorrect.

Reply:
Thank you all for the feedback. The latest code should address the issues. In my testing the Annotation header is visually hidden when the empty state is shown, but visible when it's not. @yyassi-heartex @bmartel @hlomzik

Reply:
Ideally this annotation ID should exist in a different place that makes more sense on the application, so that it can be picked up by selenium tests directly and opening the history panel is avoided (this to me seems like a workaround that unfortunately adds unnecessary steps/delay to the tests).

Reply:
Oh I understand now, this change was to the **Empty State** version of this panel, not the general space. Ok, what you had originally would have been correct and passed selenium tests. Sorry I was confused about the change. 🙏

Reply:
No worries!